src/modules/Authentictation/Components/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,7 +1,148 @@
+      })
+      .catch((error) => {
+        console.log(error);
+        toast.error(
+          error?.response?.data?.message ||
+            "something went wrong please try again"
+        );
+      });
+  };
 
-export default function ForgetPassword() {
   return (
-    <div>ForgetPassword</div>
-  )
-}
+    <>
+      <Box
+        component="div"
+        sx={{
+          ml: { md: "7rem", xs: "2rem" },
+          mt: { md: "3.9rem", xs: "2rem" },
+        }}>
+        <Typography component="h1" sx={{ fontSize: "1.87rem" }}>
+          Forgot password
+        </Typography>
+
+        <Box component="div" mt="22px">
+          <Typography component="p">
+            If you already have an account register
+            <Typography component="span" display={"block"}>
+              You can{" "}
+              <Box
+                ml={1}
+                sx={{ color: red[600] }}
+                fontWeight={600}
+                component={"span"}>
+                <Link className="formLinks" to="/login">
+                  Login here !
+                </Link>
+              </Box>
+            </Typography>
+          </Typography>
+        </Box>
+
+        <Box component="div" mt={"96px"}>
+          <form onSubmit={handleSubmit(onSubmitHandler)}>
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "column",
+                width: { md: "70%", xs: "90%" },
+              }}>
+              <label htmlFor="email">Email</label>
+              <TextField
+                sx={{
+                  "& .MuiFilledInput-root": {
+                    "&:before": { borderBottom: "none" },
+                    "&:hover:not(.Mui-disabled):before": {
+                      borderBottom: "none",
+                    },
+                    "&:after": { borderBottom: "none" },
+                  },
+
+                  backgroundColor: "#F5F6F8",
+                }}
+                hiddenLabel
+                defaultValue=""
+                variant="filled"
+                size="small"
+                placeholder="Please type here ..."
+                {...register("email", validationRules.email)}
+              />
+
+{errors.email && (
+                <FormHelperText
+                  sx={{
+                    color: "#EB5148",
+                    paddingBottom: "0.3rem",
+                    fontWeight: "bold",
+                  }}>
+                  {errors?.email?.message}
+                </FormHelperText>
+              )}
+            </Box>
+            <Button
+              disabled={isSubmitting}
+              type="submit"
+              sx={{
+                mt: { md: "63px", xs: "2rem" },
+                backgroundColor: "#3252DF",
+                color: "white",
+                width: "70%",
+                py: "0.8rem",
+                textTransform: "none",
+                "&.Mui-disabled": {
+                  background: "#949fcf",
```

Comment:
unfortunately,  fixed colors will not change when switching between MUI themes. I suggest using closer colors from the MUI design system, this will enable changing the theme effortlessly without the need to update the colors ourselves.